### DISCUSSION TITLE
Fancy hyphens on man page rendering hurt search

### DISCUSSION THREAD

**Discussion by jidanni** [curl/curl#11643] [General]
I can't search curl man pages, using e.g., "-jar" in various pagers,
due to the fancy formatting.
```
$ man curl|perl -nwle 'print for /--cookie\W+jar/g'|cat -vt|sort -u
--cookieM-bM-^@M-^Pjar
```
I would just use an ASCII "-", like on the cat(1) man page.
```
$ man cat|perl -nwle 'print for /--squeeze\W+blank/g'|cat -vt|sort -u
--squeeze-blank
```

Seen on Debian.

**Comment by bagder** [comment]
This is not a "discussion", this is an issue. But also already fixed in git.
